genie_registry/clinical.py

```diff
@@ -175,6 +175,37 @@ def _check_int_year_consistency(
     return ""
 
 
+def _check_year_death_validity(clinicaldf: pd.DataFrame) -> str:
+    """
+    Check if YEAR_DEATH >= YEAR_CONTACT.
+    YEAR_DEATH should alway be equal or large than YEAR_CONTACT if they are both available.
+    This function compares rows with numeric values in both columns and returns comparion results("True"/"False")
```

Comment:
Nit: I think this statement here is a bit confusing because it says that the function itself is returning true/false/'N/A' values? 

I think the statement is helpful to be added in comments down in your function below above the relevant sections of code because it's not obvious what the code is doing, and the statement clears that up